### PR TITLE
Added question in general cs questions of an inplace sorting algorithm

### PR DESCRIPTION
**Pull request by PranjalSK03**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/general-cs-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,13 @@
 const generalCSQuiz = [
+  {
+    Question: "Which of the following is not an inplace sorting algorithm?",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:3]:
```suggestion
    Question: "Which of the following is NOT an in-place sorting algorithm?",
```

File: src/data/general-cs-quiz.ts [hunk 2]
```diff
@@ -1,4 +1,13 @@
...
+    Answer: "Merge Sort",
+    Distractor1: "Bubble sort",
+    Distractor2: "Insertion sort",
+    Distractor3: "Quick sort",
+    Explanation:"An inplace sorting algo uses no auxiliary space to process output",
```
Comment (jdwilkin4) [inline src/data/general-cs-quiz.ts:8]:
```suggestion
    Explanation:"Merge sort uses the Divide and Conquer paradigm and is not considered an in-place sorting algorithm.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/general-cs-quiz.ts (hunk 1 above)
Comment (PranjalSK03) [reply src/data/general-cs-quiz.ts:3]: ok i will do

File: src/data/general-cs-quiz.ts (hunk 2 above)
Comment (PranjalSK03) [reply src/data/general-cs-quiz.ts:8]: ok i am on it

**Review by PranjalSK03 — COMMENTED** [review]
made the changes ig

**Comment by PranjalSK03** [comment]
can you please check i have made the required changes

**Pull request merged**